Common/include/CConfig.hpp

```diff
@@ -437,6 +437,20 @@ class CConfig {
   Max_DeltaTime,               /*!< \brief Max delta time. */
   Unst_CFL;                    /*!< \brief Unsteady CFL number. */
 
+  /* Gradient smoothing options */
+  bool SmoothGradient;            /*!< \brief Flag for enabling gradient smoothing. */
+  su2double SmoothingEps1;        /*!< \brief Parameter for the identity part in gradient smoothing. */
+  su2double SmoothingEps2;        /*!< \brief Parameter for the Laplace part in gradient smoothing. */
```

Comment:
It's good practice to group data types (reduces the size of structs/classes).
```suggestion
  su2double SmoothingEps1;        /*!< \brief Parameter for the identity part in gradient smoothing. */
  su2double SmoothingEps2;        /*!< \brief Parameter for the Laplace part in gradient smoothing. */
  bool SmoothGradient;            /*!< \brief Flag for enabling gradient smoothing. */
```